libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -1216,3 +1216,18 @@ AP_GPS_UBLOX::broadcast_configuration_failure_reason(void) const {
         }
     }
 }
+
+void AP_GPS_UBLOX::Write_DataFlash_Log_Startup_messages() const
+{
+    AP_GPS_Backend::Write_DataFlash_Log_Startup_messages();
+    if (is_configured()) {
```

Comment:
@OXINARF Huh.  I inverted that just before pushing up, as I was getting all-message-unconfigured and thought I'd misread the meaning of is_configured.

@WickedShell I'll remove these messages - I'd misunderstood what is_configured meant.